dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3PutJob.java

```diff
@@ -174,23 +176,44 @@ public void executeJob() throws Exception {
                 final boolean isFilePropertiesEnable = settings.getFilePropertiesSettings().getFilePropertiesEnable();
                 if (isFilePropertiesEnable) {
                     job.withMetadata(new MetadataAccessImpl(fileMapper));
+                    // Path file = fileMapper.get(filename);
+                    // check whether chunk are available
+                    job.attachWaitingForChunksListener(retryAfterSeconds -> {
+                        for (int retryTimeRemaining = retryAfterSeconds; retryTimeRemaining >= 0; retryTimeRemaining--) {
+                            try {
+                                updateMessage("No available chunks to transfer. Trying again in " + retryTimeRemaining + "seconds");
+                                Thread.sleep(1000);
+                            } catch (final Exception e) {
+                                LOG.error("Exception in attachWaitingForChunksListener" + e.getMessage());
```

Comment:
This log statement should not just print the exception message, it should also pass in the full exception so that way the stack trace can be logged.  So, this should be updated to look like:

```java

LOG.error("Exception in attachWaitingForChunksListener: " + e.getMessage(), e);
```